packaging/ubuntu-16.04/tests/integrationtests

```diff
@@ -46,11 +46,11 @@ systemctl daemon-reload
 systemctl restart snapd
 
 # Spread will only buid with recent go
-snap install --classic --channel 1.17/stable go
+snap install --classic go
 
 # and now run spread against localhost
 export GOPATH=/tmp/go
-/snap/bin/go get -u github.com/snapcore/spread/cmd/spread
+/snap/bin/go install github.com/snapcore/spread/cmd/spread@latest
```

Comment:
Does this work? Sorry, I'm not sure how to run that test so I can't try it myself but I thought the `@latest` syntax didn't work with `go install` in go1.18

Reply:
I think it does, here is what I did:
```
$ cd /tmp/
$ go version
go version go1.18 linux/amd64
$ export GOPATH=/tmp/go
$ go install github.com/snapcore/spread/cmd/spread@latest
go: downloading github.com/snapcore/spread v0.0.0-20211206135717-b426a3f04d80
go: downloading github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e
go: downloading golang.org/x/net v0.0.0-20210716203947-853a461950ff
go: downloading golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
go: downloading golang.org/x/oauth2 v0.0.0-20210628180205-a41e5a781914
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading github.com/kr/text v0.1.0
go: downloading cloud.google.com/go v0.65.0
go: downloading golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
go: downloading golang.org/x/sys v0.0.0-20210615035016-665e8c7367d1
$ ls /tmp/go/bin/
spread
```
it does not work with go1.13 and probably also anything below go 1.17 (but I have no validated this). This should be fine because a few lines above we install `snap install --classic go` which will give us the current go version 1.18+

Or am I missing something?

Reply:
No, that seems right so I must've misremembered. Thanks for confirming it